### PR TITLE
Basic CIFAR10 Backpropagation Model

### PR DESCRIPTION
**Pull request by ghost**
Implemented the architecture of the convolutional network used in the feedback alignment paper(based on the architecture in the dropout paper referenced therein) by modifying the tensorflow cifar10 tutorial code. Learning rate had to be reduced, but no values seem to be given in either paper so it was set to 0.001. Preprocessing wasn't changed and feedback alignment is yet to be implemented.

### CODE REVIEW

**Comment by Tamini** [comment]
Did you make any changes from CIFAR10 besides the learning rate? Or is the rest of it straight from the tutorial?

**Comment by ghost** [comment]
Sorry about the late response. The only difference should be the learning rate(and the folder in which it saves), which was simply what I found worked (somehow slipped my mind that the hyperparameters would be in the Lua source code, so we should go with whatever is presented there). The architecture is different and a maxnorm constraint was placed on all the variables as well, but I assume you mean other than that.
